src/include/peloton_main/peloton_main.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "common/thread_pool.h"
+#include "network/network_manager.h"
+#include "concurrency/epoch_manager.h"
+#include "gc/gc_manager.h"
+
+namespace peloton {
+
+//===--------------------------------------------------------------------===//
+// Global Setup and Teardown
+//===--------------------------------------------------------------------===//
+
+class PelotonMain {
```

Comment:
Would there be any reason why we could not make this class be just "Peloton"? Peloton sounds better over PelotonMain.

Reply:
Or just `PelotonDB` or `Database`?